src/coreclr/ilasm/prebuilt/asmparse.cpp

```diff
@@ -5813,47 +5813,47 @@ yyparse (void)
 
   case 404: /* ddItem: INT64_ '(' int64 ')' ddItemCount  */
 #line 1071 "asmparse.y"
-                                                             { __int64* p = new (nothrow) __int64[(yyvsp[0].int32)];
+                                                             { int64_t* p = new (nothrow) int64_t[(yyvsp[0].int32)];
                                                                if(p != NULL) {
                                                                  for(int i=0; i<(yyvsp[0].int32); i++) p[i] = *((yyvsp[-2].int64));
-                                                                 PASM->EmitData(p, sizeof(__int64)*(yyvsp[0].int32)); delete (yyvsp[-2].int64); delete [] p;
+                                                                 PASM->EmitData(p, sizeof(int64_t)*(yyvsp[0].int32)); delete (yyvsp[-2].int64); delete [] p;
                                                                } else PASM->report->error("Out of memory emitting data block %d bytes\n",
-                                                                     sizeof(__int64)*(yyvsp[0].int32)); }
+                                                                     sizeof(int64_t)*(yyvsp[0].int32)); }
 #line 5823 "asmparse.cpp"
     break;
 
   case 405: /* ddItem: INT32_ '(' int32 ')' ddItemCount  */
 #line 1078 "asmparse.y"
-                                                             { __int32* p = new (nothrow) __int32[(yyvsp[0].int32)];
+                                                             { int32_t* p = new (nothrow) int32_t[(yyvsp[0].int32)];
                                                                if(p != NULL) {
                                                                  for(int i=0; i<(yyvsp[0].int32); i++) p[i] = (yyvsp[-2].int32);
-                                                                 PASM->EmitData(p, sizeof(__int32)*(yyvsp[0].int32)); delete [] p;
+                                                                 PASM->EmitData(p, sizeof(int32_t)*(yyvsp[0].int32)); delete [] p;
```

Comment:
I know this is pre-existing, but cases like this with no space before the `*` operator make this hard to visually parse

Reply:
Yes, the ilasm/ildasm code has many issues (the formatting is not the worst part). I would be best to rewrite it in C#...